src/Types/Post.php

```diff
@@ -83,16 +90,20 @@ public function modify_query_input_fields( array $query_args ) : array {
 
 				$post_ids = $connected->get_posts();
```

Comment:
( I know this isnt part of the PR, but this is a performance nightmare, since youre hitting the database an extra time for each possible connection, on top of in the resolver itself.
Im not familiar enough with posts-2-posts to suggest a solve, but surely theres a way to query posts by connection meta, instead of querying posts by connections in order to then query posts by id. )

Reply:
I am not sure how to do it either. I have tried to looked at the P2P docs but there does not seem to be a way from my reading other than this. Remember this was all written by Kellen in the beginning who knows MUCH more than I with PHP